### ISSUE TITLE
Delete P3 hack

### ISSUE THREAD

**Issue by barbeque-squared** [UltraStar-Deluxe/USDX#746]
USDX currently implements a 'feature' that when it encounters a P3 tag, it's as if you had it in both P1 and P2. It will never actually get saved back though, it's able to parse it but that's it.

I've also never actually seen this used so I think it's time to remove it. Depending on how it handles the situation when the tag is still used and when it logs/crashes about it, I might postpone this until after #697.

See this discussion: https://github.com/UltraStar-Deluxe/format/issues/17

**Comment by basisbit** [comment]
I suggest only going forward with doing this once a consensus / new format is defined. Currently P3 tag is supported and it gets used by people/communities. This would be a breaking change.
